README.md

```diff
@@ -47,10 +45,7 @@ An example of a denied admission would be:
 
 An example of no policy matched:
 1. If the image does not match against any policy
-1. Fallback to [deprecated policy-controller validation behavior](#deprecated-policy-controller-validation-behavior)
-1. Validation will be attempted against the secret defined under `cosign.secretKeyRef.name` during helm installation.
-  1. If a valid signature or attestation is obtained, image is admitted
-  1. If no valid signature or attestation is obtained, image is denied
+1. The image is admitted
```

Comment:
Would you mind just updating this section to be bold and say something like:
```
If no policy matches an image, the image is admitted
```

so it's crystal clear :)

Reply:
Yes, I didn't pushed these changes. 
Done now!